source/packages/nova.json

```diff
@@ -19,7 +19,7 @@
     "Algeria",
     "All resources loaded.",
     "American Samoa",
-    "An error occured while uploading the file.",
+    "An error occurred while uploading the file.",
```

Comment:
It's a Laravel Nova 3.30.0 (latest release):

![image](https://user-images.githubusercontent.com/10347617/149946429-5da69a91-4dfc-4d6b-9501-90ab04f67ea1.png)